### ISSUE TITLE
Contact form accessibility audit 

### ISSUE THREAD

**Issue by kellyChex** [samvera/hyrax#3963]
### Descriptive summary

In testing for the Hyrax 3.0 release, the following two points are actions that need to be taken on the Hyrax contact form page in order to be more accessible:

**Ensures landmarks are unique:**

![Screen Shot 2019-09-11 at 4 38 17 PM](https://user-images.githubusercontent.com/9621070/64742880-9cb6be80-d4b2-11e9-8088-982a65db3f82.png)

**All page content must be contained by landmarks:**

![Screen Shot 2019-09-11 at 4 40 51 PM](https://user-images.githubusercontent.com/9621070/64742963-f1f2d000-d4b2-11e9-84be-c692a6b0f17c.png)

### Steps to reproduce the behavior

1. Install the Axe plugin or addon from Deque (https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/ is the firefox version).
2. Open https://nurax-dev.curationexperts.com/contact?locale=en
3. Open the inspector (right-click, and select "Inspect element")
4. Click on the 'Axe' tab
5. Click 'Analyze'

### Tab Test 

In Firefox and Safari, unable to hit the send button. 

Chrome works. 

![Screen Recording 2019-09-16 at 01 50 PM](https://user-images.githubusercontent.com/9621070/64992543-f3dcda80-d888-11e9-8f36-a13ee64c6113.gif)

**Comment by dlim87** [comment]
Tabbing is working. I'm still seeing the 2nd accessibility concern: 
![image](https://user-images.githubusercontent.com/5492162/68429080-48eaed80-0162-11ea-8988-12e11d0d199a.png)

**Comment by dlim87** [comment]
This should probably be fixed with #4156

**Comment by kellyChex** [comment]
The 'All page content must be contained by landmarks' is still a concern.

**Comment by bkeese** [comment]
Everything here seems to be good in base hyrax. 
The landmarks concern is specific to the Nurax test page footer.
There, `<div class="blue-footer container">` should probably be changed to something like `<footer class="blue-footer container" aria-label="blue Nurax footer">`

**Comment by MariaStarPower** [comment]
This **passes**, but with some minor issues.

Results of the Contact Form Accessibility audit:

<details>

![image](https://user-images.githubusercontent.com/32848057/100948201-30134f80-34bc-11eb-8b10-72564e5cc9ac.png)

![image](https://user-images.githubusercontent.com/32848057/100948241-46211000-34bc-11eb-8816-ca3716674952.png)

![image](https://user-images.githubusercontent.com/32848057/100948294-6781fc00-34bc-11eb-97a4-3d1d50669dc4.png)

![image](https://user-images.githubusercontent.com/32848057/100948353-88e2e800-34bc-11eb-9745-6766ed27dab4.png)

</details>
